nntrainer/src/tensor.cpp

```diff
@@ -672,38 +685,38 @@ Tensor Tensor::normalization() const {
 }
 
 Tensor Tensor::standardization() const {
-  Tensor result(batch, height, width);
+  Tensor result(dim.batch(), dim.height(), dim.width());
 
-  for (int k = 0; k < batch; ++k) {
-    int K = k * height * width;
+  for (unsigned int k = 0; k < dim.batch(); ++k) {
+    int K = k * dim.height() * dim.width();
```

Comment:
I think it would be better to change `K` (and other variables in for loops) 's type as well.

Reply:
fixed.